exporters/sql_export/README.rst

```diff
@@ -0,0 +1,281 @@
+*************************************
+Eksport til Actual State SQL database
+*************************************
+
+Indledning
+==========
+Denne eksport laver et dags-dato udtræk af MO og afleverer det i en SQL database.
+
+For at opnå den nødvendige afviklingshastighed, tilgår eksporten data direkte fra
+LoRa hvor det er muligt at lave bulk udtræk, baseret på den kendte datamodel for
+OS2MO behandles de udtrukne data så SQL eksporten for et udseende som svarer til
+det man finder i MO.
+
+Tabellerne er for den praktiske anvendeligheds skyld ikke 100% normaliserede, der
+vil således for alle klasser altid være både en reference til primærnøglen for
```

Comment:
vil for eksempel for klassers vedkommende, når de optræder i andre tabeller, være både en reference til...